testing/load-test/v2/common.js

```diff
@@ -0,0 +1,229 @@
+import {check} from 'k6';
+import http from 'k6/http';
+import {Counter, Trend} from 'k6/metrics';
+
+import {randomIntBetween} from "https://jslib.k6.io/k6-utils/1.0.0/index.js";
+
+export let randomByteString = function (size) {
```

Comment:
maybe this function should be a level up, since you will probably want it when v3 gets produce.